Header header
float64[] joint_positions  # Position for each joint
float64[] joint_velocities # Velocity for each joint 
float64[] joint_accelerations # Acceleration for each joint
geometry_msgs/Pose pose    # End-effector pose corresponding to joint positions 